### PR TITLE
[Anomali] - Added new command and enhance reputation commands

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-3269)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
image

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Anomali_ThreatStream/ReleaseNotes/2_0_13.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Anomali ThreatStream v3
+- Added the new command: **threatstream-search-intelligence**.
+- Added the new argument **threat_model_association** to the reputation commands.
```
Comment (dantavori) [inline Packs/Anomali_ThreatStream/ReleaseNotes/2_0_13.md:5]:
```suggestion
- Added a new command: ***threatstream-search-intelligence***.
- Added the ***threat_model_association*** argument to the ***ip***, ***file***, ***domain*** and ***url*** reputation commands.
```

File: Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py [hunk 2]
```diff
@@ -381,6 +437,26 @@ def parse_indicators_list(iocs_list):
     return iocs_context
 
 
+def parse_intelligence_list(intelligence_list):
```
Comment (dantavori) [inline Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py:440]: no need to parse the response

File: Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py [hunk 3]
```diff
@@ -1177,6 +1362,49 @@ def get_indicators(client: Client, **kwargs):
...
+        while len(intelligence_context) < limit:
+            offset += len(intelligence_list)
+            kwargs['limit'] = limit
+            kwargs['offset'] = offset
+            intelligence_list = client.http_request("GET", url, params=kwargs).get('objects', None)
+            if intelligence_list:
+                intelligence_context.extend(parse_intelligence_list(intelligence_list))
+            else:
+                break
+
```
Comment (dantavori) [inline Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py:1397]: as we talked
```suggestion
```

File: Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py [hunk 4]
```diff
@@ -1177,6 +1362,49 @@ def get_indicators(client: Client, **kwargs):
     )
 
 
+def search_intelligence(client: Client, **kwargs):
+    """
+        Returns filtered indicators by parameters from ThreatStream.
+        By default the limit of indicators result is set to 50.
+    """
+    page = kwargs['page'] = int(kwargs.get('page', 0))
+    page_size = kwargs['page_size'] = int(kwargs.get('page_size', 0))
```
Comment (dantavori) [inline Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py:1371]: don't pass these args

**Review by dantavori — CHANGES_REQUESTED** [review]

**Review by dantavori — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@MLainer1 Please regenerate the README file.
@dantavori Doc review completed.

**Pull request merged**